src/main/java/com/ca/apim/gateway/cagatewayconfig/util/gateway/BundleElementNames.java

```diff
@@ -0,0 +1,29 @@
+package com.ca.apim.gateway.cagatewayconfig.util.gateway;
+
+/**
+ * Utility class to hold element names used in gateway bundles.
+ */
+public interface BundleElementNames {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Move constants to a class or enum. ([squid:S1214](https://sonarcloud.io/organizations/ca-api-gateway/rules#rule_key=squid%3AS1214))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=com.ca.apim.gateway%3Agateway-developer-plugin&pullRequest=62&issues=AWVkEUg3pttE6qk1f6NI&open=AWVkEUg3pttE6qk1f6NI)

[ISSUE_KEY]: AWVkEUg3pttE6qk1f6NI